From 27db6f7d42214b32ce32794d300a11eb313df166 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 10 Aug 2021 10:47:04 +0300
Subject: [PATCH] [SW-0] gaudi: security: close tpc7 qm gap

Include the gap between TPC7 QM and the CFG high gap (0x7FFD000000) in
the CFG high gap closure.
So the CFG_HIGH_GAP_BASE is now starts right after TPC7 QM.

Change-Id: Id3f3c3ab402c5b5b668c19a734c8f0d15c5f4be4
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi/security.c b/arch/gaudi/security.c
index 4f4ed3d..030ecbc 100644
--- a/arch/gaudi/security.c
+++ b/arch/gaudi/security.c
@@ -1500,7 +1500,12 @@
 	LBW_RR_DBG_ETR,
 };
 
-#define CFG_HIGH_GAP_BASE	0x7FFD000000ull
+/*
+ * The CFG high gap starts at 0x7FFD000000, but there is additional one between
+ * the TPC7 QM and the CFG high gap which needs to be included as well.
+ * Include the TPC7 gap inside the CFG high gap
+ */
+#define CFG_HIGH_GAP_BASE	0x7FFCFC9000ull
 #define DBG_BASE		mmMME_S_ROM_TABLE_BASE
 #define HOST_PHYS_BASE		0x8000000000ull
 
